### ISSUE TITLE
Chunks of Flesh

### ISSUE THREAD

**Issue by xpdatabase11** [studio-minus/ppg-bugs-and-requests#1375]
**Feature description**
Being able to hack of junks of flesh which would fall off the bone to the floor.
The head would be able to explode into several pieces when exposed to huge damage.
The limbs could be cut off in more places.

**Feature purpose**
Will add a lot of realism and make it more satisfying when using higher power weapons.

**Additional context**
The changes to the gore shader in the newest version are awesome. But this would take it to the next level.

**Comment by PossiblyRemember** [comment]
I feel like this would be a cool mix with zooi's per pixel destruction, with dynamic cutting and destruction, and in all would be pretty cool, but probably hard to implement

**Comment by DindinYT37** [comment]
> I feel like this would be a cool mix with zooi's per pixel destruction, with dynamic cutting and destruction, and in all would be pretty cool, but probably hard to implement

The main reason it's not implemented is that it's too hard on performance, at least as far as I know.

**Comment by PossiblyRemember** [comment]
Agreed, but maybe have it with a toggle that's disabled by default.
